src/Controllers/EntityBrowserListBuilder.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity_browser\Entity\Controllers\EntityBrowserListBuilder.
+ */
+
+namespace Drupal\content_entity_example\Entity\Controller;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityListBuilder;
+
+
+/**
+ * Provides a list controller for entity browser.
+ *
+ * @ingroup entity_browser
+ */
+class EntityBrowserListBuilder extends EntityListBuilder {
```

Comment:
List builder needs to be added to EntityBrowser annotation. 

I get this error when trying to navigate to /admin/config/content/entity_browser/list:

```
Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException: The "entity_browser" entity type did not specify a list_builder handler. in Drupal\Core\Entity\EntityTypeManager->getHandler() (line 238 of /var/www/core/lib/Drupal/Core/Entity/EntityTypeManager.php).
```